docs/src/main/asciidoc/kafka-reactive-getting-started.adoc

```diff
@@ -0,0 +1,365 @@
+
+The `process` method is called for every Kafka _record_ from the `prices` topic (configured in the application configuration).
+Every result is sent to the `my-data-stream` in-memory stream.
+
+== The price resource
+
+Let's bind our stream to a JAX-RS resource and send events to clients.
+Create the `src/main/java/org/acme/kafka/PriceResource.java` file with the following content:
+
+[source,java]
+----
+package org.acme.kafka;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.reactive.messaging.Channel;
+import org.reactivestreams.Publisher;
+
+/**
+ * A simple resource retrieving the prices topic and sending the items to a server sent event.
+ */
+@Path("/prices")
+public class PriceResource {
+
+    @Inject
+    @Channel("my-data-stream") Publisher<Price> prices; // <1>
+
+    @GET
+    @Path("/stream")
+    @Produces(MediaType.SERVER_SENT_EVENTS) // <2>
+    public Publisher<Price> stream() { // <3>
+        return prices;
+    }
+}
+----
+<1> Injects the `my-data-stream` channel using the `@Channel` qualifier
+<2> Indicates that the content is sent using `Server Sent Events`
+<3> Returns the stream (_Reactive Stream_)
+
+== JSON serialization via Jackson
+
+Finally, we will configure JSON serialization for messages using Jackson.
+Quarkus provides default implementations for Kafka serializer/deserializer pairs using Jackson `ObjectMapper`.
+`ObjectMapperSerializer` can be used to serialize all pojos via Jackson.
+For the corresponding deserializer class, we need to create a subclass of `ObjectMapperDeserializer`.
+
+So, let's create `src/main/java/org/acme/kafka/PriceDeserializer.java`
+
+[source,java]
+----
+package org.acme.kafka;
+
+import io.quarkus.kafka.client.serialization.ObjectMapperDeserializer;
+
+public class PriceDeserializer extends ObjectMapperDeserializer<Price> {
+    public PriceDeserializer() {
+        // pass the class to the parent.
+        super(Price.class);
+    }
+}
+----
+
+[TIP]
+====
+.Message serialization in Kafka
+In this example we used Jackson to serialize/deserialize Kafka messages.
+For more options on message serialization see <<kafka-serialization>>.
+
+We strongly suggest adopting a contract-first approach using a schema registry.
+To learn more about how to use Apache Kafka with the schema registry and Avro follow the link:kafka-schema-registry-avro.adoc[Using Apache Kafka with Schema Registry and Avro] guide.
+====
+
+== Configuring the Kafka connector
+
+We need to configure the Kafka connector.
+This is done in the `application.properties` file.
+The keys are structured as follows:
+
+`mp.messaging.[outgoing|incoming].{channel-name}.property=value`
+
+The `channel-name` segment must match the value set in the `@Incoming` and `@Outgoing` annotation:
+
+* `generated-price` -> sink in which we write the prices
```

Comment:
I would remove sink and source. They seem to be leftovers from the "Fluid" times (that predated RM)